definitions/ext-shelled_spa/definition.yml

```diff
@@ -0,0 +1,39 @@
+domain: EXT
+type: SHELLED_SPA
+synthesis:
+  rules:
+
+    - identifier: shellAppName
+      name: shellAppName
+      encodeIdentifierInGUID: true
+      conditions:
+      - attribute: eventType
+        value: ShelledBrowserInteraction
+
+    - identifier: shellAppName
+      name: shellAppName
+      encodeIdentifierInGUID: true
+      conditions:
+      - attribute: eventType
+        value: ShelledPageViewTiming
+
+    - identifier: shellAppName
+      name: shellAppName
+      encodeIdentifierInGUID: true
+      conditions:
+      - attribute: eventType
+        value: ShelledJavaScriptError
+
+  tags:
+    category:
+      multiValue: false
+    appId:
+      multiValue: false
```

Comment:
In all examples i have saw this value is `0`

are we really sure it will be unique per entity? if not please remove the multiValue:false line